playbooks/openshift-node/private/config.yml

```diff
@@ -19,8 +19,6 @@
 
 - import_playbook: additional_config.yml
 
-- import_playbook: manage_node.yml
```

Comment:
This playbook should be skipping during bootstrap mode, if it isn't, that's a bug in this playbook.